site/docs/v1/tech/apis/system.adoc

```diff
@@ -302,7 +307,18 @@ link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:lock[type
 --
 
 === Response
-The JSON response from the Status API is complex and subject to change. The specific contents of the JSON body are not documented here. If you choose to use this API for monitoring purposes you should primarily use the response code to indicate server health. If you receive a `200` you may consider FusionAuth in a healthy state. The response body is intended for use by FusionAuth support.
+The JSON response from the Status API is complex and subject to change excepting the `version` key.
```

Comment:
"change excepting the `version` key." => "change. The only exception is the `version` key."